### DISCUSSION TITLE
peppy i have question

### DISCUSSION THREAD

**Discussion by rXelelo** [ppy/osu#29268] [Q&A]
Can skin makers edit betmap choosing menu?
![image](https://github.com/user-attachments/assets/7d4e3275-f500-4de7-a240-33a80802b74e)

**Comment by Avelia-tr** [comment]
yeah and no, you can add images and things like that but you can't change the texture of the elements. It's something that is planned to be upgraded if I'm not wrong.

**Comment by kstefanowicz** [comment]
Note that there's a full redesign of the song select menu (#22204) in development. Adding skinnable elements to that screen would most likely come after the redesign is finished.
